spk/tt-rss/src/conf/resource

```diff
@@ -1,14 +1,79 @@
 {
-    "php56": {
-		"php-ini": [{
-			"relpath": "etc/php/conf.d/com.synocommunity.tt-rss.ini",
-			"destpath": "/usr/local/etc/php56/conf.d"
-		}]
-    },
-	"php70": {
-		"php-ini": [{
-			"relpath": "etc/php/conf.d/com.synocommunity.tt-rss.ini",
-			"destpath": "/usr/local/etc/php70/conf.d"
-		}]
-    }
+  "mariadb10-db": {
+    "migrate-db": {
+      "flag": "{{wizard_run_migration}}",
+      "m5-db-name": "ttrss",
+      "m10-db-name": "ttrss",
+      "db-collision": "replace"
+    },
+    "admin-account-m10": "root",
+    "admin-pw-m10": "{{wizard_mysql_password_root}}",
+    "admin-account-m5": "root",
+    "admin-pw-m5": "{{wizard_mariadb5_password_root}}",
+    "create-db": {
+      "flag": "{{wizard_create_db}}",  
+      "db-name": "ttrss",
+      "db-collision": "skip"
+    },
+    "grant-user": {
+      "flag": "{{mysql_grant_user}}",
+      "db-name": "ttrss",
+      "user-name": "ttrss",
+      "host": "localhost",
+      "user-pw": "{{wizard_mysql_password_ttrss}}"
+    },
+    "drop-db-inst": {
+        "flag": "{{wizard_run_migration}}",
+        "ver": "m5",
+        "db-name": "ttrss"
+    },
+    "drop-db-uninst": true,
+    "drop-user-uninst": true
+  },
+  "webservice": {
+    "services": [{
+      "service": "tt-rss",
+      "display_name": "Tiny Tiny RSS",
+      "type": "apache_php",
+      "root": "tt-rss",
+      "backend": 1,
```

Comment:
can we use apache 2.4 instead of 2.2?
```suggestion
      "backend": 2,
```

Reply:
If we go that path it means having different requirements depending on whether DSM 6 or not (currently it's only depending on the 2.2 version).

If that's an accepted approach I could do that.

Reply:
Wait, actually apache 2.2 is not even listed in the dependencies... That wouldn't be changing anything.